djangocms_blog/admin.py

```diff
@@ -82,6 +136,14 @@ class PostAdmin(PlaceholderAdminMixin, FrontendEditableAdminMixin,
     search_fields = ('translations__title',)
     date_hierarchy = 'date_published'
     raw_id_fields = ['author']
+    actions = [
```

Comment:
I'd like to have the actions as method of the admin class as they are not going to be reused elsewhere